docs/databags.md

```diff
@@ -58,7 +62,7 @@ An Indexes Hash is part of a plaintext data bag item that defines the set of ind
 
 Roles Hash
 ----------
-A Roles Hash is a contextual (see above) Hash, part of a plaintext data bag item that defines roles for every node in a cluster, and is pointed to by the `node[:splunk][:config][:roles]` attribute (usually set in your environment).
+A Roles Hash is a contextual (see above) Hash, part of a plaintext data bag item that defines roles for every node in a cluster, and is pointed to by the `node[:splunk][:config][:roles]` attribute (usually set in your environment). A special key of 'shc_configs' is used for managing the roles on the search heads in a search head cluster.
```

Comment:
minor, All the other keys match roles of nodes (search_head, license_master, etc.) so I think search_head_cluster would make more sense and be more consistent for the special key in all of these contextual hashes.